TS_TSX.md

```diff
@@ -0,0 +1,48 @@
+# File Metrics for TypeScript (TS) and TypeScript JSX (TSX)
+
+This document outlines the definition of file metrics for TypeScript (TS) and TypeScript JSX (TSX).
+
+### 1. classes
+
+The "classes" metric counts:
+
+-   keywords like: `class`, `abstract class`, `interface`, `enum`
```

Comment:
Is it correct to say that we really count the keyword? Technically, we do not, as we count the nodes `class_declaration` and not the (unnamed) node that contains the keyword itself. 
Practically, I don't know if the keywords `class`, etc. can be only used to define a class in TS or also on other places, so that that your documentation is right anyway or not. For C and C++, this would be clearly wrong, as we can do things like 
````C++
#include <iostream>

class class_name;

class class_name {
    public:
        void print_hello() {
            std::cout << "Hello!" << std::endl;
        }
};

int main() {
    class class_name* object = new class_name();
    class class_name object2();
    class class_name object3();

    object->print_hello();
    object2.print_hello();
    object3.print_hello();
}
````

Here, we actually do only count the thing with the second class keyword.

Reply:
Also, what does "keywords like" mean? Do we also count other keywords that are similar? Then we should mention them here.

I would not write here that we count keywords at all, however. We should write that we count class declarations, abstract class declarations, interface declarations and enum declarations.